MediaBrowser.MediaEncoding/Subtitles/SubtitleEncoder.cs

```diff
@@ -94,16 +94,16 @@ public class SubtitleEncoder : ISubtitleEncoder
             return ms;
         }
 
-        private void FilterEvents(SubtitleTrackInfo track, long startPositionTicks, long? endTimeTicks, bool preserveTimestamps)
+        private void FilterEvents(SubtitleTrackInfo track, long startPositionTicks, long endTimeTicks, bool preserveTimestamps)
         {
             // Drop subs that are earlier than what we're looking for
             track.TrackEvents = track.TrackEvents
                 .SkipWhile(i => (i.StartPositionTicks - startPositionTicks) < 0 || (i.EndPositionTicks - startPositionTicks) < 0)
                 .ToArray();
 
-            if (endTimeTicks.HasValue)
+            if (endTimeTicks > 0)
             {
-                long endTime = endTimeTicks.Value;
+                long endTime = endTimeTicks;
```

Comment:
The `endTime` variable is redundant now that `endTimeTicks` is no longer `long?`. Please remove it.

Reply:
Yes. I guess I wasn't being super critical when I submitted this PR lol.